cog/daily_charge.py

```
@@ -23,95 +25,293 @@
-        await interaction.response.send_message(embed=self.embed, ephemeral=True)
+    @staticmethod
+    def is_cross_day(last_charge: datetime, executed_at: datetime):
+        assert executed_at >= last_charge
+        return executed_at.date() - last_charge.date() > timedelta(days=1)
 
-    @discord.slash_command(name="charge", description="每日充電")
-    async def charge(self, interaction):
-        user_id = interaction.user.id
-        connection, cursor = link_sql()  # SQL 會話
+    @staticmethod
+    def is_already_charged(last_charge: datetime, executed_at: datetime):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `is_already_charged`

(ANN205)